### PR TITLE
Add support for empty command documents

### LINKED ISSUE #1000: Add support for empty documents
It is possible to pass empty document to command document. One of it's usages as replacement document in update operation.

- [ ] Check for this issue number being mentioned in source code
- [ ] Check that validation function passes such documents
- [ ] Check `findAndModify` command
- [ ] Enable tests

Example: when you pass an empty `bson.D` for update with replacement document, the result we store should be the doc with only `_id` field being set.

### PR DESCRIPTION
**Pull request by w84thesun**
# Description

Closes #1000.

## Readiness checklist

* [x] I added tests for new functionality or bugfixes.
* [ ] I ran `task all`, and it passed.
* [ ] I added/updated comments for both exported and unexported top-level declarations (functions, types, etc).
* [ ] I checked comments rendering with `task godocs`.
* [x] I ensured that the title is good enough for the changelog.
* [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Assignee, Labels, Project and project's Sprint fields.
* [x] I marked all done items in this checklist.

### CODE REVIEW

**Comment by noisersup** [comment]
Tests are still failing, because it's needed to change some fields there, but please check if it's suitable for our requirements

File: integration/update_test.go [hunk 1]
```diff
@@ -228,15 +228,6 @@ func TestUpdateReplaceDocuments(t *testing.T) {
 			expectedFilter: bson.D{{"_id", "document-composite"}},
 			expected:       bson.D{{"_id", "document-composite"}, {"replacement-value", int32(1)}},
 		},
-		// TODO: https://github.com/FerretDB/FerretDB/issues/1000
```
Comment (w84thesun) [inline integration/update_test.go:231]: This test is basically the same as "ReplaceEmptyDocument" in update_compat_test.go

File: internal/handlers/pg/msg_update.go [hunk 2]
```diff
@@ -68,6 +68,7 @@ func (h *Handler) MsgUpdate(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg,
 	}
 
 	var created bool
+
```
Comment (AlekSi) [inline internal/handlers/pg/msg_update.go:71]: All changes in this file are whitespace changes that also make this file different from Tigris version. Why we need them?

Comment (w84thesun) [reply internal/handlers/pg/msg_update.go]: Restored as it was. `MsgUpdate` and `MsgFindAndModify` are a huge mess. I think we should refactor them.

**Review by noisersup — APPROVED** [review]

**Review by rumyantseva — APPROVED** [review]
LGTM

**Pull request merged**